ee/modules/030-cloud-provider-openstack/hooks/migrate/migrate_terraform_state.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2023 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+// TODO remove after 1.48 release
+
+package hooks
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/deckhouse/deckhouse/go_lib/dependency"
+	"github.com/deckhouse/deckhouse/go_lib/dependency/k8s"
+)
+
+const TerraformStateDataKey = "node-tf-state.json"
+const TerraformStateNamespace = "d8-system"
+const OpenstackV2ResourceType = "openstack_blockstorage_volume_v2"
+const OpenstackV3ResourceType = "openstack_blockstorage_volume_v3"
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	OnStartup: &go_hook.OrderedConfig{Order: 10},
+}, dependency.WithExternalDependencies(openstackTerraformStateMigration))
+
+func openstackTerraformStateMigration(input *go_hook.HookInput, dc dependency.Container) error {
+	input.LogEntry.Infof("Starting Terraform migrate hook")
```

Comment:
this is unnecessary

Reply:
I've removed this line: `input.LogEntry.Infof("Starting Terraform migrate hook")`